src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/MethodTypeInference.cs

```diff
@@ -551,10 +549,9 @@ private void MakeExplicitParameterTypeInferences(Binder binder, BoundExpression
                 !MakeExplicitParameterTypeInferences(binder, (BoundTupleLiteral)argument, target, kind, ref useSiteDiagnostics))
             {
                 // Either the argument is not a tuple literal, or we were unable to do the inference from its elements, let's try to infer from argument type
-                if (IsReallyAType(source))
+                if (IsReallyAType(argument.Type))
                 {
-                    var annotation = GetNullableAnnotation(argument);
-                    ExactOrBoundsInference(kind, TypeSymbolWithAnnotations.Create(source, annotation), target, ref useSiteDiagnostics);
+                    ExactOrBoundsInference(kind, GetTypeWithAnnotations(argument), target, ref useSiteDiagnostics);
```

Comment:
>ExactOrBoundsInference [](start = 20, length = 22)

:memo: inside `ExactOrBoundsInference` tuples may get exploded into parts (see `LowerBoundTupleInference`). So we're applying `AsSpeakable` in-depth here.